packages/mural-picker/src/components/mural-picker/index.tsx

```diff
@@ -47,6 +48,7 @@ export type Slots = {
 
 interface PropTypes {
   apiClient: ApiClient;
+  getRoomsByWorkspacePageSize?: number;
```

Comment:
**concern**: IDK if the page size should be a prop set on the actual component, or a property that we could be set on the api client side, or event simply defined as a constant in each context.

I don't really see a good reason why we should let the end-user alter this value.